tsconfig.test.json

```diff
@@ -2,6 +2,9 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "importHelpers": false,
-    "target": "es2018"
+    "target": "es2018",
+    // workaround for a bug in ts-jest where moduleResolution gets overwritten
```

Comment:
I made this comment originally thinking it was a .js file but it seems that ts-jest doesn't mind these comments in the  json file. Unorthodox, but I thought it was worth it to keep the comments